pkgs/by-name/ge/geteduroam-cli/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  makeWrapper,
+  versionCheckHook,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "geteduroam-cli";
+  version = "0.4";
+
+  src = fetchFromGitHub {
+    owner = "geteduroam";
+    repo = "linux-app";
+    tag = version;
+    hash = "sha256-9+IjrHg536/6ulj94CBhYWY0S3aNA7Ne4JQynMmsLxE=";
+  };
+
+  vendorHash = "sha256-9SNjOC59wcEkxJqBXsgYClHKGH7OFWk3t/wMPLANAy0=";
+
+  nativeBuildInputs = [
+    makeWrapper
```

Comment:
I would use `makeBinaryWrapper` but it's up to you to change.

Reply:
In fact it's dead code from my tests with the GUI, sorry about that